packages/vitest/src/node/test-run.ts

```diff
@@ -118,7 +123,8 @@ export class TestRun {
     await Promise.all(finishedTestCases.map(testCase => this.vitest.report('onTestCaseFinished', testCase)))
     await Promise.all(finishedTestModules.map(module => this.vitest.report('onTestModuleFinished', module)))
 
-    await Promise.all(queuedTestModules.map(module => this.vitest.report('onTestModuleQueued', module)))
+    // TODO: Not needed as RPC has onQueued already. Do we want to remove that and centralize resolving here instead?
```

Comment:
queued state cannot be reported in `onTaskUpdate` because the task is not created yet, so we always have to use `onQueued`

Reply:
Makes sense, I'll update this part. 👍